content/components.html

```diff
@@ -356,66 +356,59 @@ <h2>Neomad profiles</h2>
   </section>
 
   <section class=st-section>
-    <div class=st-title>Article in profil list</div>
+    <div class=st-title>Articles in profile list</div>
 
     <div class="st-container st-full">
-      <article class=preview>
-        <div class=bar>&nbsp;</div>
-        <div class=bar>&nbsp;</div>
-        <div class=bar>&nbsp;</div>
-        <div class=bar>&nbsp;</div>
-        <a href>Write this article</a>
-      </article>
-    </div>
+      <section class=previews-list>
```

Comment:
I'm still not good with the class naming but WDYT about `previews` or `preview-list`?

Reply:
I saw we had used a class `.article-list`, hence the name

Reply:
yes but `article-list` not `articles-list`, WDYT?

Reply:
It actually doesn't matter to me. Which do you prefer?

Reply:
`.preview-list` just to keep consistance ^^